kectl/app/cmd/util/edgecoreinstaller.go

```diff
@@ -0,0 +1,216 @@
+               return err
+       }
+
+       //Add edge node in api-server using kubectl command
+       //kubectl apply -f $GOPATH/src/github.com/kubeedge/kubeedge/build/node.json -s http://192.168.20.50:8080
+       nodeJSONApply := fmt.Sprintf("kubectl apply -f %s -s http://%s", KubeEdgeConfigNodeJSON, ku.K8SApiServerIP)
+       cmd := &Command{Cmd: exec.Command("sh", "-c", nodeJSONApply)}
+       err = cmd.ExecuteCmdShowOutput()
+       errout := cmd.GetStdErr()
+       if err != nil || errout != "" {
+           return fmt.Errorf("%s", errout)
+       }
+       fmt.Println(cmd.GetStdOutput())
+
+       return nil
+}
+
+//modifyEdgeYamlNodeJSON modifies the edge config files
+func (ku *KubeEdgeInstTool) modifyEdgeYamlNodeJSON() error {
+
+	//Update edge.yaml, server ip for websocket communication
+	edgeYaml, err := ioutil.ReadFile(KubeEdgeConfigEdgeYaml)
+	if err != nil {
+		return err
+	}
+
+	serverIPAddr := strings.Split(ku.K8SApiServerIP, ":")[0]
+	rep1 := bytes.Replace(edgeYaml, []byte(KubeEdgeToReplaceKey2), []byte(serverIPAddr), -1)
+
+	//Update edge.yaml, node ip to act as node id
+	//Get node ip address from the interface
+	edgeIP, err := GetInterfaceIP()
+	if err != nil {
+		return err
+	}
+	rep2 := bytes.Replace(rep1, []byte(KubeEdgeToReplaceKey1), []byte(edgeIP), -1)
+
+	if err = ioutil.WriteFile(KubeEdgeConfigEdgeYaml, rep2, 0666); err != nil {
+		return err
+	}
+
+	switch ku.ToolVersion {
+	//These are the older version of KubeEdge, Hence creating node.json file directly,
+	//since it is not available in ../edge/conf path.
+	//From higher release, it shall be placed and the default condition will get executed.
+	case "0.1", "0.2", "0.2.1":
+		//Update node.json, node ip to act as node id
+		rep := bytes.Replace([]byte(KubeEdgeNodeJSONContent), []byte(KubeEdgeToReplaceKey1), []byte(edgeIP), -1)
+
+		if err = ioutil.WriteFile(KubeEdgeConfigNodeJSON, rep, 0666); err != nil {
+			return err
+		}
+	default:
+		nodeJSON, err := ioutil.ReadFile(KubeEdgeConfigNodeJSON)
+		if err != nil {
+			return err
+		}
+
+		//Update node.json, node ip to act as node id
+		rep := bytes.Replace(nodeJSON, []byte(KubeEdgeToReplaceKey1), []byte(edgeIP), -1)
+
+		if err = ioutil.WriteFile(KubeEdgeConfigNodeJSON, rep, 0666); err != nil {
+			return err
+		}
+	}
+
+	//Add edge node in api-server using kubectl command
+	//kubectl apply -f $GOPATH/src/github.com/kubeedge/kubeedge/build/node.json -s http://192.168.20.50:8080
+	nodeJSONApply := fmt.Sprintf("kubectl apply -f %s -s http://%s", KubeEdgeConfigNodeJSON, ku.K8SApiServerIP)
+	cmd := &Command{Cmd: exec.Command("sh", "-c", nodeJSONApply)}
+	err = cmd.ExecuteCmdShowOutput()
+	errout := cmd.GetStdErr()
+	if err != nil || errout != "" {
+		return fmt.Errorf("%s", errout)
+	}
+	fmt.Println(cmd.GetStdOutput())
+
+	return nil
+}
+
+//TearDown method will remove the edge node from api-server and stop edge_core process
+func (ku *KubeEdgeInstTool) TearDown() error {
+
```

Comment:
please remove blank lines at the beginning and end of functions
Applicable to all similar places